### ISSUE TITLE
MyBatis

### ISSUE THREAD

**Issue by azzahrah** [bwajtr/java-persistence-frameworks-comparison#19]
My be you must update Mybatis, cause many improvement current release....

**Comment by bwajtr** [comment]
Yeah, this whole project would need an update, but I had no spare time recently :(
But thanks for bringing it up... I'll try to find some time
